File: src/lib/models/DTO/Domain/PaymentInfoDto.cs
using System;

namespace DTO.Domain
{
    public class PaymentInfoDto : BaseDomainDto
    {
        public string CardNumber{ get; set; }
        public DateTime ExpirationDate{ get; set; }
        public string Cvc{ get; set; }
        public short CardTypeId{ get; set; }
        public CardTypeDto CardType{ get; set; } 
    }
}

File: src/ui/asq-ui/src/app/_models/domain/payment-info-dto.ts
import { BaseDomainDto } from './base-domain-dto';
import { CardTypeDto } from './card-type-dto';
import * as moment from 'moment';

export class PaymentInfoDto extends BaseDomainDto{
    public cardTypeId: number = null;
    public cardNumber: string = null;
    public cvc: string = null;
    public expirationDate: moment.Moment = null;
    public expirationDateModel: string = null;

    public CardType: CardTypeDto = null;

    constructor(){ super(); }
}
